Makefile

```diff
@@ -879,20 +878,14 @@ release-sources: | $(DIST_DIRS)
 
 .PHONY: release-docs
 release-docs: | $(DIST_DIRS) docs
-	tar -czf $(DIST)/release/gitea-docs-$(VERSION).tar.gz -C ./docs/public .
+	tar -czf $(DIST)/release/gitea-docs-$(VERSION).tar.gz -C ./docs .
```

Comment:
So this is docs source but not the docs. It's a break change from before.

Reply:
Why is that breaking?
For whom?
People who self-host our documentation?

Reply:
As I mentioned above, a follow-up could be to add some form of conversion to HTML/PDF or similar.